openapi/components/schemas/Quote.yaml

```
@@ -176,27 +213,43 @@ properties:
               description: Unit price of the invoice item.
               type: number
               format: double
+              nullable: true
             quantity:
               description: Quantity of the invoice item.
               type: integer
-            price:
-              description: Total price of the invoice item.
+            period:
+              description: Date interval of the invoice item.
+              type: string
+              nullable: true
+            setupUnitPrice:
+              description: Unit price of the invoice item.
               type: number
               format: double
-              readOnly: true
-            periodStartTime:
-              description: Date and time when the billing period starts.
-              type: string
-              format: date-time
-            periodEndTime:
-              description: Date and time when the billing period ends.
+              nullable: true
+            trialUnitPrice:
+              description: Unit price of the invoice item.
+              type: number
+              format: double
+              nullable: true
+            trialPeriod:
+              description: Date interval of the invoice item trial.
               type: string
-              format: date-time
-            tax:
+              nullable: true
+            taxAmount:
               description: Invoice item tax.
-              readOnly: true
-              allOf:
-                - $ref: ./Invoices/InvoiceTaxItem.yaml
+              type: number
+              format: double
+              nullable: true
+            setupTaxAmount:
+              description: Invoice setup item tax.
+              type: number
+              format: double
+              nullable: true
+            trialTaxAmount:
+              description: Invoice trial item tax.
```

nitpick: [noun stack](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#noun-strings-or-noun-stacks)

I'm not sure what these field represent. I made an effort to expand on them, not sure if they are correct.

```suggestion
              description: Tax amount of the setup that is displayed on the invoice.
              type: number
              format: double
              nullable: true
            trialTaxAmount:
              description: Tax amount of the trial that is displayed on the invoice.
```